services.py

```
@@ -0,0 +1,368 @@
+    @staticmethod
+    def add_log(
+        session_id: int,
+        message: str,
+        is_user: bool,
+        response_time: Optional[float] = None,
+        token_count: Optional[int] = None
+    ) -> ConversationLog:
+        """会話ログを追加"""
+        try:
+            # セッションの存在確認
+            session = SessionService.get_session_by_id(session_id)
```

Ruff (0.12.2)

Local variable `session` is assigned to but never used

Remove assignment to unused variable `session`

(F841)